src/pybind/mgr/cephadm/module.py

```diff
@@ -818,30 +818,33 @@ def _get_security_config(self) -> Tuple[bool, bool, bool]:
         security_enabled = self.secure_monitoring_stack or mgmt_gw_enabled
         return security_enabled, mgmt_gw_enabled, oauth2_proxy_enabled
 
-    def get_mgmt_gw_internal_endpoint(self) -> Optional[str]:
+    def _get_mgmt_gw_endpoint(self, is_internal: bool) -> Optional[str]:
```

Comment:
@adk3798 these functions used to be part of the `mgmt_gateway.py` but since the introduction of the`oauth2-proxy` which needs to call the `get_mgmt_gw_external_endpoint` I moved them to the module.py to avoid circular dependencies between mgmt_gateway and oauth2_proxy services.